### ISSUE TITLE
HOME env variable not set during startup

### ISSUE THREAD

**Issue by mmarinaccio** [phusion/passenger-docker#344]
Hi there -

I've been struggling with an issue over the last few days related to the `HOME` environment variable allegedly not being set.

I'm using this image for multiple purposes at my organization, including hosting API routes, background job processing, and scheduled tasks.

For scheduled tasks, I need to run a one-off Ruby command that results in the container shutting down after completion. The docs recommend doing this: `/sbin/my_init -- COMMAND ARGS`.

One of my example commands looks like this:
```bash
/sbin/my_init -- bundle exec bin/rails runner -e ${environment} "'${rails_command}'"
```

However, this returns the below error:
<details>
  <summary> ArgumentError: non-absolute home</summary>
	2022-04-13T16:48:26.938-04:00	ArgumentError: non-absolute home

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/pry-0.12.2/lib/pry/pry_class.rb:5:in `expand_path'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/pry-0.12.2/lib/pry/pry_class.rb:5:in `<class:Pry>'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/pry-0.12.2/lib/pry/pry_class.rb:1:in `<top (required)>'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/pry-0.12.2/lib/pry.rb:119:in `<top (required)>'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bundler-2.1.4/lib/bundler/runtime.rb:74:in `block (2 levels) in require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bundler-2.1.4/lib/bundler/runtime.rb:69:in `each'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bundler-2.1.4/lib/bundler/runtime.rb:69:in `block in require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `each'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bundler-2.1.4/lib/bundler.rb:174:in `require'

	2022-04-13T16:48:26.938-04:00	/home/app/loyalsnap/config/application.rb:7:in `<top (required)>'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/railties-5.2.6/lib/rails/command/actions.rb:22:in `require_application!'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/railties-5.2.6/lib/rails/command/actions.rb:14:in `require_application_and_environment!'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/railties-5.2.6/lib/rails/commands/runner/runner_command.rb:29:in `perform'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/railties-5.2.6/lib/rails/command/base.rb:69:in `perform'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/railties-5.2.6/lib/rails/command.rb:46:in `invoke'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/railties-5.2.6/lib/rails/commands.rb:18:in `<top (required)>'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'

	2022-04-13T16:48:26.938-04:00	/usr/local/rvm/gems/ruby-2.5.9/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'

	2022-04-13T16:48:26.938-04:00	bin/rails:8:in `<top (required)>'
</details>

I've run into this with rake tasks, too. For example, when launching this container as a background job worker, I run something like the below:
```bash
    exec chpst -u app env \
        HOME=/home/app \
        RAILS_ENV=${RAILS_ENV} \
        QUEUE=* \
        RESQUE_TERM_TIMEOUT=8 \
        RESQUE_PRE_SHUTDOWN_TIMEOUT=120 \
        TERM_CHILD=1 \
        /home/app/org/bin/rake resque:work
```

This fails with the same `ArgumentError: non-absolute home` error if I don't include `HOME=/home/app` in the command.

I tried doing this with the original command as shown below, but it doesn't seem to make a difference:
```bash
HOME=/home/app /sbin/my_init -- bundle exec bin/rails runner -e ${environment} "'${rails_command}'"
```

I'm also setting `ENV HOME /home/app` (also tried /root, as documented) in my dockerfile, but this doesn't work either.

Any suggestions would be hugely appreciated.

TLDR: `HOME` isn't set during the docker boot process and it's causing some of my Rails commands to fail.

Thank you!
